### PR TITLE
change time interval to 5 second

### PR DESCRIPTION
**Pull request by Shwetajain148**
In bulk mode, Logs wait for 20000 milliseconds (20 seconds) to send logs to the loggly.
When application exit within 20 second after sending the log, it doesn't reach to the loggly.

We should decrease wait time interval to overcome the above issue.

### CODE REVIEW

**Comment by Shwetajain148** [comment]
@mchaudhary Please review this PR.

**Comment by Shwetajain148** [comment]
@mchaudhary I have added a log4net-loggly.nuspec file in the code and changed the bulk mode log collection time to 5 seconds. Kindly review and merge the PR.

**Comment by Shwetajain148** [comment]
@mchaudhary In this PR, I have added the log4net-loggly.nuspec file that will be helpful to publish new NuGet package in future, and have changed log collection time in bulk mode. Kindly review and merge this PR.

cc: @mostlyjason @psquickitprageet

File: source/log4net-loggly/log4net-loggly.nuspec [hunk 1]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<package >
+  <metadata>
+    <id>log4net-loggly</id>
+    <version>7.2.2</version>
```
Comment (psquickitprageet) [inline source/log4net-loggly/log4net-loggly.nuspec]: change to 7.2.3.

**Comment by Shwetajain148** [comment]
Accidently I had removed my fork's branch. Due to which, this PR was closed.

I have raised a new PR #2.

**Pull request closed without merging**